db/migrations/20181210092417-createEmailAutomationTable.js

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  up: (queryInterface, DataTypes) => queryInterface.createTable('emailAutomation', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: DataTypes.INTEGER,
+    },
+    status: {
+      type: DataTypes.ENUM,
+      values: ['success', 'failure'],
+      allowNull: false,
+    },
+    error: {
```

Comment:
@IAMOTZ , You can change this column to statusMessage so that we can have a message even for a successful automation

Reply:
The is true @psibomana 
I would update the column.